fe1-web/model/network/method/message/data/election/SetupElection.ts

```diff
@@ -0,0 +1,124 @@
+import {
+  Hash, Timestamp, Lao, EventTags,
+} from 'model/objects';
+import { OpenedLaoStore } from 'store';
+import { ProtocolError } from 'model/network/ProtocolError';
+import { validateDataObject } from 'model/network/validation';
+import { Question } from 'model/objects/Election';
+import { ActionType, MessageData, ObjectType } from '../MessageData';
+import { checkTimestampStaleness } from '../Checker';
+
+export class SetupElection implements MessageData {
+  public readonly object: ObjectType = ObjectType.ELECTION;
+
+  public readonly action: ActionType = ActionType.SETUP;
+
+  public readonly lao: string;
```

Comment:
According to [the protocol](https://github.com/dedis/student_21_pop/pull/265/files), this field is the lao id. Thus, if I'm not wrong, the type should be a `Hash` like just below

Reply:
Yes that's right. It is changed now :)